src/views/Project/index.jsx

```diff
@@ -79,22 +152,44 @@ export default class Project extends Component {
           'state:open',
         ].join(' ');
 
-        return this.fetch(searchQuery);
+        return this.fetchGithub(searchQuery);
+      })
+    );
+    const variable = {
+      search: {
+        tags: ['good-first-bug'],
+        statuses: ['NEW', 'UNCONFIRMED', 'ASSIGNED', 'REOPENED'],
+      },
+      paging: {
```

Comment:
The `paging` field will never change so we should probably just add it directly in `fetchMore`.